docs/source/daml/code-snippets/Reference.daml

```diff
@@ -2,6 +2,13 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 {-# OPTIONS -Wno-controller-can #-}
+
+-- template-let warning disabled because 'start template let snippet' is
+-- used by docs.daml.com for older compiler versions and to explain
+-- new deprecation warning in 2.8.0
+-- https://github.com/digital-asset/daml/issues/17263
+{-# OPTIONS_GHC -Wno-template-let #-}
```

Comment:
It seems that `docs.daml.com` uses a single version of this (and other `docs/**/*.daml` files), so I'm not sure what we're going to do when we fully remove this feature so the docs for older versions stay truthful. I guess we can just move the example into a comment, but ew.

Reply:
Oof, good point... I guess we can cross that bridge when we get to it. Is there anything about our current approach that will make that difficult do we think?